.github/workflows/integration_tests.yml

```diff
@@ -48,3 +48,7 @@ jobs:
           fal run --profiles-dir .
           echo "*** Checking fal output files ***"
           python check_outputs.py
+          echo "*** Starting fal run with experimental ordering ***"
+          fal run --experimental-ordering --profiles-dir .
```

Comment:
Wouldn't we want to change the code then so that it changes the ordering a bit?

Reply:
yea we can keep this to test it doenst error. but we can add more integration tests for the ordering